rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -471,28 +499,30 @@ def _download_kubeconfig(ocm_cmnd, cluster_id, my_path, type):
 
 
 def _download_cluster_admin_kubeconfig(rosa_cmnd, cluster_name, my_path):
+    cluster_admin_create_time = int(time.time())
     return_data = {}
-    logging.info('Creating cluster-admin user on cluster %s (100 retries allowed)' % cluster_name)
+    logging.info('Creating cluster-admin user on cluster %s (30 minutes timeout)' % cluster_name)
     kubeconfig_cmd = [rosa_cmnd,  "create", "admin", "-c", cluster_name, "-o", "json"]
     logging.debug(kubeconfig_cmd)
-    cluster_admin_create_time = int(time.time())
-    for trying in range(1, 101):
+    # Waiting 30 minutes for cluster-admin user to be created
+    while datetime.datetime.utcnow().timestamp() < cluster_admin_create_time + 30 * 60:
         if force_terminate:
             logging.error("Exiting cluster access process for %s cluster after capturing Ctrl-C" % cluster_name)
             return return_data
         process = subprocess.Popen(kubeconfig_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=my_path, universal_newlines=True)
         stdout, stderr = process.communicate()
         if process.returncode != 0:
-            logging.warning('Try: %d. Failed to create cluster-admin user on %s with this stdout/stderr:' % (trying, cluster_name))
+            logging.warning('Failed to create cluster-admin user on %s with this stdout/stderr:' % cluster_name)
             logging.warning(stdout)
             logging.warning(stderr)
-            logging.error('Try: %d. Waiting 5 seconds for the next try on %s' % (trying, cluster_name))
+            logging.error('Waiting 5 seconds for the next try on %s until %s' % (cluster_name, datetime.datetime.fromtimestamp(cluster_admin_create_time + 30 * 60)))
```

Comment:
shouldnt this be logging.debug? it's not an "error" that its not ready yet

Reply:
changed to warning